src/tools/_tools.font-size.scss

```diff
@@ -95,16 +102,23 @@
         // Just output the passed in font-size value.
         font-size: $font-size $important;
         @if ($line-height == auto) {
-            // If the font-size is custom, just output the
-            // `$FLUIDMS-GLOBAL-LINE-HEIGHT` as `em` value, so we can use `calc`
-            // with it when a line-height modifier is used.
-            $_line-height: $FLUIDMS-GLOBAL-LINE-HEIGHT * 1em;
-            // If the line-height is modified, add/subtract that from the
-            // calculated line-height.
-            @if $line-height-modifier {
-                $_line-height: calc(#{$_line-height} + #{$line-height-modifier * $FLUIDMS-GLOBAL-BASELINE});
+            @if $FLUIDMS-GLOBAL-BASELINE {
+                // If the font-size is custom, just output the
+                // `$FLUIDMS-GLOBAL-LINE-HEIGHT` as `em` value, so we can use `calc`
+                // with it when a line-height modifier is used.
+                $_line-height: $FLUIDMS-GLOBAL-LINE-HEIGHT * 1em;
+                // If the line-height is modified, add/subtract that from the
+                // calculated line-height.
+                @if $line-height-modifier {
+                    $_line-height: calc(#{$_line-height} + #{$line-height-modifier * $FLUIDMS-GLOBAL-BASELINE});
+                }
+                line-height: $_line-height $important;
+            }
+            @if (not $FLUIDMS-GLOBAL-BASELINE) {
```

Comment:
duplication (l91-95)

Reply:
there is still a duplication or am I missing something?

Reply:
@fuhlig How would you eliminate that duplication?